src/rules/no-global-function-names/__tests__/index.js

```diff
@@ -0,0 +1,51 @@
+import rule, { ruleName, messages } from "..";
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+  syntax: "scss",
+
+  accept: [
+    {
+      code: `
+      @use color;
+      a {
+       background: color.adjust(#6b717f, $red: 15);
+      }
+    `,
+      description: "Non-global function with different name"
+    },
+    {
+      code: `
+      @use color;
+      a {
+       background: color.red(#6b717f);
+      }
+    `,
+      description: "Non-global function with different name"
+    }
+  ],
```

Comment:
I'd also add accept tests for using a global function that's still allowed (such as `rgb()` and/or `if()`).